<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[the name of homePageis failed]]>
    </line>
  </reporter-output>
  <suite started-at="2023-01-27T16:03:31 IST" name="Default suite" finished-at="2023-01-27T16:03:46 IST" duration-ms="14955">
    <groups>
    </groups>
    <test started-at="2023-01-27T16:03:31 IST" name="Default test" finished-at="2023-01-27T16:03:46 IST" duration-ms="14955">
      <class name="getScreenShotTestNG.ActiTimeTestNGLogin">
        <test-method is-config="true" signature="setUp()[pri:0, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:31 IST" name="setUp" finished-at="2023-01-27T16:03:34 IST" duration-ms="2724" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginPage()[pri:0, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:34 IST" name="loginPage" finished-at="2023-01-27T16:03:34 IST" duration-ms="360" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:34 IST" name="tearDown" finished-at="2023-01-27T16:03:36 IST" duration-ms="1751" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:36 IST" name="setUp" finished-at="2023-01-27T16:03:38 IST" duration-ms="1732" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="homePage()[pri:1, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:38 IST" name="homePage" finished-at="2023-01-27T16:03:40 IST" duration-ms="2263" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [actiTIME - nter Time-Track] but found [actiTIME - Enter Time-Track]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [actiTIME - nter Time-Track] but found [actiTIME - Enter Time-Track]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at getScreenShotTestNG.ActiTimeTestNGLogin.homePage(ActiTimeTestNGLogin.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[the name of homePageis failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- homePage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:getScreenShotTestNG.ActiTimeTestNGLogin@5fd4f8f5]" started-at="2023-01-27T16:03:40 IST" name="tearDown" finished-at="2023-01-27T16:03:46 IST" duration-ms="5879" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- getScreenShotTestNG.ActiTimeTestNGLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
